string                    posture_name
string[]                  selected_groups
meka_posture_recorder/PostureRecordWaypoint[]  waypoints
